src/crackmes/hackthebox/find-the-easy-pass/fire-up-the-debugger.md

```diff
@@ -0,0 +1,81 @@
+# Fire up the debugger
+
+The next step will be to put a breakpoint at 0x0045412b, press `F2` key

```

Comment:
Same issue here, why at that address?

Reply:
I explained why this breakpoint with the last changes.